[2024-04-10T00:53:32.683+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: upload_congress_data.bq_create_bills_Congress_partitioned_table_task manual__2024-04-10T00:36:19.141018+00:00 [queued]>
[2024-04-10T00:53:32.705+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: upload_congress_data.bq_create_bills_Congress_partitioned_table_task manual__2024-04-10T00:36:19.141018+00:00 [queued]>
[2024-04-10T00:53:32.705+0000] {taskinstance.py:2193} INFO - Starting attempt 8 of 10
[2024-04-10T00:53:32.727+0000] {taskinstance.py:2217} INFO - Executing <Task(BigQueryInsertJobOperator): bq_create_bills_Congress_partitioned_table_task> on 2024-04-10 00:36:19.141018+00:00
[2024-04-10T00:53:32.749+0000] {standard_task_runner.py:60} INFO - Started process 3584 to run task
[2024-04-10T00:53:32.758+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'upload_congress_data', 'bq_create_bills_Congress_partitioned_table_task', 'manual__2024-04-10T00:36:19.141018+00:00', '--job-id', '33', '--raw', '--subdir', 'DAGS_FOLDER/upload_congress_data.py', '--cfg-path', '/tmp/tmp1w0q4txk']
[2024-04-10T00:53:32.764+0000] {standard_task_runner.py:88} INFO - Job 33: Subtask bq_create_bills_Congress_partitioned_table_task
[2024-04-10T00:53:32.831+0000] {task_command.py:423} INFO - Running <TaskInstance: upload_congress_data.bq_create_bills_Congress_partitioned_table_task manual__2024-04-10T00:36:19.141018+00:00 [running]> on host f811dc6f69ec
[2024-04-10T00:53:33.128+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='***@***.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='upload_congress_data' AIRFLOW_CTX_TASK_ID='bq_create_bills_Congress_partitioned_table_task' AIRFLOW_CTX_EXECUTION_DATE='2024-04-10T00:36:19.141018+00:00' AIRFLOW_CTX_TRY_NUMBER='8' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-04-10T00:36:19.141018+00:00'
[2024-04-10T00:53:33.137+0000] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2024-04-10T00:53:33.138+0000] {connection.py:269} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2024-04-10T00:53:33.141+0000] {base.py:83} INFO - Using connection ID 'google_cloud_default' for task execution.
[2024-04-10T00:53:33.270+0000] {bigquery.py:2811} INFO - Executing: {'query': {'query': 'CREATE OR REPLACE TABLE Congress.bill_status         PARTITION BY latestActionDate_formatted         AS         SELECT *, DATE(bill.latestAction.actionDate) latestActionDate_formatted         FROM Congress.bill_status_external_table;', 'useLegacySql': False}}'
[2024-04-10T00:53:33.284+0000] {bigquery.py:1613} INFO - Inserting job ***_upload_congress_data_bq_create_bills_Congress_partitioned_table_task_2024_04_10T00_36_19_141018_00_00_fd4b3b2031d0bb3f882b86b1482c15cb
[2024-04-10T00:53:35.332+0000] {taskinstance.py:2731} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 444, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 414, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/providers/google/cloud/operators/bigquery.py", line 2870, in execute
    job.result(timeout=self.result_timeout, retry=self.result_retry)
  File "/home/airflow/.local/lib/python3.11/site-packages/google/cloud/bigquery/job/query.py", line 1595, in result
    do_get_result()
  File "/home/airflow/.local/lib/python3.11/site-packages/google/cloud/bigquery/job/query.py", line 1584, in do_get_result
    super(QueryJob, self).result(retry=retry, timeout=timeout)
  File "/home/airflow/.local/lib/python3.11/site-packages/google/cloud/bigquery/job/base.py", line 971, in result
    return super(_AsyncJob, self).result(timeout=timeout, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/google/api_core/future/polling.py", line 261, in result
    raise self._exception
google.api_core.exceptions.BadRequest: 400 Invalid date: '2024-03-18T20:19:24Z' Field: updateDate; Value: 2024-03-18T20:19:24Z; reason: invalidQuery, location: query, message: Invalid date: '2024-03-18T20:19:24Z' Field: updateDate; Value: 2024-03-18T20:19:24Z; reason: invalid, message: Error while reading table: Congress.bill_status_external_table, error message: JSON processing encountered too many errors, giving up. Rows: 1; errors: 1; max bad: 0; error percent: 0

Location: US
Job ID: airflow_upload_congress_data_bq_create_bills_Congress_partitioned_table_task_2024_04_10T00_36_19_141018_00_00_fd4b3b2031d0bb3f882b86b1482c15cb

[2024-04-10T00:53:35.360+0000] {taskinstance.py:1149} INFO - Marking task as UP_FOR_RETRY. dag_id=upload_congress_data, task_id=bq_create_bills_Congress_partitioned_table_task, execution_date=20240410T003619, start_date=20240410T005332, end_date=20240410T005335
[2024-04-10T00:53:35.377+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 33 for task bq_create_bills_Congress_partitioned_table_task (400 Invalid date: '2024-03-18T20:19:24Z' Field: updateDate; Value: 2024-03-18T20:19:24Z; reason: invalidQuery, location: query, message: Invalid date: '2024-03-18T20:19:24Z' Field: updateDate; Value: 2024-03-18T20:19:24Z; reason: invalid, message: Error while reading table: Congress.bill_status_external_table, error message: JSON processing encountered too many errors, giving up. Rows: 1; errors: 1; max bad: 0; error percent: 0

Location: US
Job ID: airflow_upload_congress_data_bq_create_bills_Congress_partitioned_table_task_2024_04_10T00_36_19_141018_00_00_fd4b3b2031d0bb3f882b86b1482c15cb
; 3584)
[2024-04-10T00:53:35.471+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-04-10T00:53:35.548+0000] {taskinstance.py:3312} INFO - 0 downstream tasks scheduled from follow-on schedule check
